blockchain/vm/interpreter.go

```diff
@@ -343,7 +305,10 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte) (ret []byte, err
 		if verifyPool {
 			verifyIntegerPool(in.intPool)
 		}
-
+		if in.evm.Config.EnableOpDebug {
+			opTime[op] += uint64(time.Since(opExecStart).Nanoseconds())
+			opCnt[op] += 1
+		}
```

Comment:
When are both variables to be reinitialized? their scope seems global. Currently there seems not.

Reply:
The intention is to accumulate all the exec time and count info while node is turned on, it is initialized when the node turns on.

Reply:
I was confused that it does not need to distinguish the execution per context.